bundler.d/libvirt.rb

```diff
@@ -1,4 +1,4 @@
 group :libvirt do
-  gem 'fog-libvirt', '>= 0.12.0'
+  gem 'fog-libvirt', '>= 0.13.1'
```

Comment:
technically, this bump is unnecessary, as Foreman itself will work just fine with both 0.12.0 and 0.13.0, it's just the tests that are broken with those versions (0.12.0 because we now rely on the new mocked values from 0.13.0, 0.13.0 because it broke the URL thing).

keeping it at 0.12.0 allows packaging to lazily catch up on this, instead of having to have 0.13.1 go in at the same time as this change to not block nightlies.

(once again: not blocking, but a strong preference ;-) )

Reply:
Good point, updated.